src/cli/cluster/run_kbn_optimizer.ts

```diff
@@ -35,6 +35,7 @@ export function runKbnOptimizer(opts: Record<string, any>, config: LegacyConfig)
     repoRoot: REPO_ROOT,
     watch: true,
```

Comment:
non related to that PR, but shouldn't that config be associated with any config param too?

Reply:
Possibly

Reply:
But I'd just as soon remove the ability to disable watching in `--dev` mode

Reply:
But I'd just as soon remove the ability to disable watching in `--dev` mode